### PR TITLE
user class ver1

### PR DESCRIPTION
**Pull request by YUAN0510**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,6 +29,12 @@
 			<artifactId>spring-boot-starter-security</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>javax.persistence</groupId>
+			<artifactId>persistence-api</artifactId>
+			<version>1.0.2</version>
+		</dependency>
+
```
Comment (dulekang1993) [inline pom.xml:37]: what is this for

Comment (dulekang1993) [reply pom.xml:37]: Got it.

Comment (YUAN0510) [reply pom.xml:37]: notation for attributes about the customer class

File: src/main/java/com/pika/Onlinegrocerystore/domain/Customer.java [hunk 2]
```diff
@@ -1,15 +1,53 @@
 package com.pika.Onlinegrocerystore.domain;
 
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.graalvm.compiler.lir.LIRInstruction;
```
Comment (dulekang1993) [inline src/main/java/com/pika/Onlinegrocerystore/domain/Customer.java:5]: What does this package do? Did you include its' dependency in the POM? I got an error in my IDEA.

Comment (dulekang1993) [inline src/main/java/com/pika/Onlinegrocerystore/domain/Customer.java:5]: What does this package do? Did you include its' dependency in the POM? I got an error in my IDEA.

Comment (dulekang1993) [reply src/main/java/com/pika/Onlinegrocerystore/domain/Customer.java:5]: I deleted it.

**Pull request merged**